docs/modules/reference/pages/billing/subscription/user-subscription.adoc

```diff
@@ -0,0 +1,119 @@
+= Manage Users
+
+Payara Cloud allows you to have multiple users accessing your subscription.
+In order to do so you invite the user into subscription via email.
+Subsequently, they would register or use their existing Payara Cloud account and get access to the subscription.
+
+== Roles
+
+We recognize multiple roles of users in Payara Cloud:
+
+Billing Manager::
+Users in this role have access to all billing and payment information related to the subscription.
+They can also invite new users to subscription and alter their roles.
+They do not necessarily have permissions to deploy applications deployed to Payara Cloud.
+
+Administrator::
+Administrators have full permissions to work deploy and configure applications deployed to Payara Cloud.
+They can also invite new user to subscription and alter their roles.
+
+Billing Manager with Administrative Privilege::
+This role is union of two roles above, granting you all the permissions.
+When you sign up to Payara Cloud, this is the role you have in your subscription.
+
+Member::
```

Comment:
Not very clear to me, can a member belong to more than one namespace within a given subscription? Also, can they belong to a given namespace across subscriptions? So I can add that here to make it clearer.

Reply:
Namespaces are not part of the equation at this point. We haven't started that feature yet.

A user may be member of multiple subscriptions.
In every subscription they have one of the four roles, where fourth role (Billing Manager with Administrative privileges) is union of Billing Manager and Administrator. This is the role the initial users gets.